plutus-core/plutus-core/src/PlutusCore/Constant/Debug.hs

```diff
@@ -22,21 +22,19 @@ type TermDebug = Term TyName Name DefaultUni DefaultFun ()
 --
 -- >>> :t enumerateDebug False
 -- enumerateDebug False :: Bool
--- >>> :t enumerateDebug $ Just 'a'
--- enumerateDebug $ Just 'a' :: Maybe Char
```

Comment:
Had to drop it, 'cause it now generates a ton of garbage instead of being stuck. I'll work on it some time later.